client/.gitignore

```diff
@@ -36,3 +36,11 @@ yarn-error.log*
 
 # typescript
 tsconfig.tsbuildinfo
+
+# components
+/pages/post-gig
+/pages/gigs
+/pages/companies
+/components/companies
+/components/post-gig
+/components/gigs
```

Comment:
What is this for?

Reply:
We're commenting out these components until we decide to work on them (we'll stick to only Jobs and Devs for now). This is my rudimentary way of doing it, do you propose a better way?

Reply:
I guess that's one way to do it but a better way, imo, is to simply not stage the files you don't want to commit. You can do this by unchecking them in the Github desktop gui, for example.

Reply:
But I don't think this is a blocker. Merging this. :)

Reply:
...as I say that, there are a million conflicts due to the Prettier PR that was merged. Could you rebase this branch against the `main` branch? If you can't, I can have a go tomorrow. It's 3am! :3